packages/office-ui-fabric-react/src/components/List/List.types.ts

```diff
@@ -68,6 +68,12 @@ export interface IListProps extends React.HTMLAttributes<List | HTMLDivElement>
    */
   onRenderCell?: (item?: any, index?: number, isScrolling?: boolean) => React.ReactNode;
 
+  /**
+   * Optional callback for monitoriing when rendering has beed completed due to change in List pages.
```

Comment:
Nit: `monitoring => monitoring` and `beed => been`, but I'd consider:

>Optional callback invoked when List has rendered.

Or something similar